sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+azure.activedirectory.tenant-id=81acb995-e5a3-408c-a42c-1e14453388dc
+azure.activedirectory.client-id=1fdfe6d0-4511-47e0-898c-ea8bec2910fb
+azure.activedirectory.client-secret=kunJ4_-JLD~CV_Epg1N5iv~HY9SSUQhHOO
+
+azure.activedirectory.authorization.graph.scopes=https://graph.microsoft.com/Calendars.Read
+azure.activedirectory.authorization.arm.scopes=https://management.core.windows.net/user_impersonation
+azure.activedirectory.authorization.arm.on-demand=true
+azure.activedirectory.authorization.office.scopes=https://manage.office.com/ActivityFeed.Read
+
+
+
+# It's suggested the logged in user should at least belong to one of the below groups
+# If not, the logged in user will not be able to access any authorization controller rest APIs
+azure.activedirectory.user-group.allowed-groups=group1, group2
+azure.activedirectory.environment=global-v2-graph
+azure.activedirectory.user-group.key=@odata.type
+azure.activedirectory.user-group.value=#microsoft.graph.group
+azure.activedirectory.user-group.object-id-key=id
```

Comment:
Above 4 properties does not exist any more, please merge master branch to your branch, then delete the 4 properties.